kratos/utilities/specifications_utilities.h

```diff
@@ -50,47 +50,17 @@ namespace
-        {"Quadrilateral3D8",                      GeometryData::KratosGeometryType::Kratos_Quadrilateral3D8},
-        {"Quadrilateral3D9",                      GeometryData::KratosGeometryType::Kratos_Quadrilateral3D9},
-        {"Tetrahedra3D10",                        GeometryData::KratosGeometryType::Kratos_Tetrahedra3D10},
-        {"Tetrahedra3D4",                         GeometryData::KratosGeometryType::Kratos_Tetrahedra3D4},
-        {"Triangle2D3",                           GeometryData::KratosGeometryType::Kratos_Triangle2D3},
-        {"Triangle2D6",                           GeometryData::KratosGeometryType::Kratos_Triangle2D6},
-        {"Triangle3D3",                           GeometryData::KratosGeometryType::Kratos_Triangle3D3},
-        {"Triangle3D6",                           GeometryData::KratosGeometryType::Kratos_Triangle3D6},
-        {"Line2D2",                               GeometryData::KratosGeometryType::Kratos_Line2D2},
-        {"Line2D3",                               GeometryData::KratosGeometryType::Kratos_Line2D3},
-        {"Line3D2",                               GeometryData::KratosGeometryType::Kratos_Line3D2},
-        {"Line3D3",                               GeometryData::KratosGeometryType::Kratos_Line3D3},
-        {"Point2D",                               GeometryData::KratosGeometryType::Kratos_Point2D},
-        {"Point3D",                               GeometryData::KratosGeometryType::Kratos_Point3D},
-        {"Sphere3D1",                             GeometryData::KratosGeometryType::Kratos_Sphere3D1},
-        {"NurbsCurve",                            GeometryData::KratosGeometryType::Kratos_Nurbs_Curve},
-        {"NurbsSurface",                          GeometryData::KratosGeometryType::Kratos_Nurbs_Surface},
-        {"NurbsVolume",                           GeometryData::KratosGeometryType::Kratos_Nurbs_Volume},
-        {"NurbsCurveOnSurface",                   GeometryData::KratosGeometryType::Kratos_Nurbs_Curve_On_Surface},
-        {"SurfaceInNurbsVolume",                  GeometryData::KratosGeometryType::Kratos_Surface_In_Nurbs_Volume},
-        {"BrepCurve",                             GeometryData::KratosGeometryType::Kratos_Brep_Curve},
-        {"BrepSurface",                           GeometryData::KratosGeometryType::Kratos_Brep_Surface},
-        {"BrepCurveOnSurface",                    GeometryData::KratosGeometryType::Kratos_Brep_Curve_On_Surface},
-        {"QuadraturePointGeometry",               GeometryData::KratosGeometryType::Kratos_Quadrature_Point_Geometry},
-        {"QuadraturePointCurveOnSurfaceGeometry", GeometryData::KratosGeometryType::Kratos_Quadrature_Point_Curve_On_Surface_Geometry},
-        {"QuadraturePointSurfaceInVolumeGeometry",GeometryData::KratosGeometryType::Kratos_Quadrature_Point_Surface_In_Volume_Geometry}
-    };
+    static std::unordered_map<std::string, GeometryData::KratosGeometryType> string_geometry_map = GenerateStringGeometryMap();
```

Comment:
Is this really useful?
Wouldn't it make more sense to just call the other fct?

Reply:
The other function gives a string from a geometry type, the map does the opposite

Reply:
ah sorry for the noise then